File: lib/jazz/generators/controller/templates/controller.js
var <%= name.capitalize.pluralize %>Controller = Jazz.Controller.create(
  { 
  
    initialize : function(){
    
    },
     
    index      : function(){
    
    },
    
    show       : function(){
    
    },
    
    make       : function(){
    
    },
    
    edit       : function(){
    
    },
    
    create     : function(){
    
    },
    
    update     : function(){
    
    },
    
    destroy    : function(){
    
    }
  }
);

<%= name.downcase.pluralize -%>_controller = new <%= name.capitalize.pluralize -%>Controller;

File: lib/jazz/generators/controller/controller.rb
module Jazz
  module Generators
    class Controller < Thor::Group
    	include Thor::Actions
    	
    	attr_accessor :name
    	
    	def self.source_root
        File.join(File.dirname(__FILE__))
      end
    	
      def create
        create_controller
      end
      
      def create_controller
        template  "templates/controller.js", "app/controllers/#{name.downcase.pluralize}_controller.js"
      end
      
    end
  end
end